index.html

```diff
@@ -62,5 +62,198 @@ <h1>Hi👋, I'm Cephas Tokoli</h1>
+                            <p>Ruby on Rails</p>
+                            <p>JavaScript</p>
+                        </div>
+                        <div class="details-button">
+                        <div>
+                            <p >See this project</p>
+                            <img src="images/Union.svg" alt="union">
+                        </div>
+                        </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="project">
+                    <div class="laptop-img">
+                        <img src="images/laptop1.png" alt="laptop">
+                    </div>
+                    <div class="project-details">
+                        <div class="tech-details">
+                            <h2 class="project-name">Project name goes here</h2>
+                        <div class="tech-stack">
+                            <p>HTML/CSS</p>
+                            <p>Ruby on Rails</p>
+                            <p>JavaScript</p>
+                        </div>
+                        <div class="details-button">
+                        <div>
+                            <p >See this project</p>
+                            <img src="images/Union.svg" alt="union">
+                        </div>
+                        </div>
+                        </div>
+                    </div>
+                </div>
+        
+                <div class="project">
+                    <div class="laptop-img">
+                        <img src="images/laptop2.png" alt="laptop">
+                    </div>
+                    <div class="project-details">
+                        <div class="tech-details">
+                            <h2 class="project-name">Project name goes here</h2>
+                        <div class="tech-stack">
+                            <p>HTML/CSS</p>
+                            <p>Ruby on Rails</p>
+                            <p>JavaScript</p>
+                        </div>
+                        <div class="details-button">
+                        <div>
+                            <p >See this project</p>
+                            <img src="images/Union.svg" alt="union">
+                        </div>
+                        </div>
+                        </div>
+                    </div>
+                </div>
+        
+                <div class="project">
+                    <div class="laptop-img">
+                        <img src="images/laptop3.png" alt="laptop">
+                    </div>
+                    <div class="project-details">
+                        <div class="tech-details">
+                            <h2 class="project-name">Project name goes here</h2>
+                        <div class="tech-stack">
+                            <p>HTML/CSS</p>
+                            <p>Ruby on Rails</p>
+                            <p>JavaScript</p>
+                        </div>
```

Comment:
As requested by the previous reviewer and according to the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) please kindly use `ul` and `li` for categories in the "Work" section.  i.e. `HTML/CSS`, `Ruby on Rails`, and `JavaScript` should be list items. 🛠